glimpse/helpers.py

```diff
@@ -1985,3 +1985,46 @@ def _parse_parallel(parallel):
     else:
         n = parallel
     return n
+
+def change_extenstion(infile,extension):
+    """
+    Changes a given file path to an identical file path but with a different extension
+    ex: image_01.JPG to image_01.JSON
+
+    Arguments:
+
+        infile (str): Input file path to be changed
+        extension (str): output file extension
+    
+    Return:
+        outfile (str) file with new extension
+    """
+    if not os.path.isfile(infile):
+        raise Exception("File {} Not Found".format(infile))
+    token = infile.split(".")[-1] #get old extension
+    outfile = infile.split(token)[0] # check for redundent extensions
+    outfile = outfile.split("/")[-1]
+    outfile += extension
+    return outfile
+
+def save_observercams(observer,directory,print_path=False):
```

Comment:
I'm not very motivated to add a helper function that is a thin wrapper around Camera.write(). However, I appreciate the motivation. Perhaps Image and Observer should have their own read/write JSON methods.

Reply:
Fair. The more comfortable I get with GLIMPSE, the less inclined I am to write redundant code and use the built-in tools.